packages/botonic-react/src/experimental/webchat-app.jsx

```diff
@@ -141,7 +141,11 @@ export class WebchatApp {
   onServiceEvent(event) {
     if (event.action === 'connectionChange')
       this.webchatRef.current.setOnline(event.online)
-    else if (event.action === 'update_message_info')
+    // TODO: Temporary solution, decide how we will send these events in next iterations
+    else if (event.message.action === 'update_message_info') {
+      const { message } = event.message
+      this.updateMessageInfo(message.id, message)
+    } else if (event.action === 'update_message_info')
```

Comment:
Should this be if be removed? Can `'update_message_info'` come from 2 different sides (`event.action` and `event.message.action`)?

Why not using `event.message.type` instead? I see that we're using multiple attributes to define the type/action and I think it's a little bit confusing, maybe would be a good idea to set them just in one attribute.